gdc_client/query/index.py

```diff
@@ -39,6 +42,32 @@ def get_access(self, uuid):
         if uuid in self.metadata.keys():
             return self.metadata[uuid]['access']
 
+    def _get_hits(self, url, metadata_query):
+        """Get hits metadata from a given API endpoint
+
+        :param str url: Endpoint URL
```

Comment:
Small thing but can you please use use the Google style of docstrings for parameters and returns? I was hoping to standardize this for the gdc-client.

https://github.com/uc-cdis/rfcs/blob/master/text/0003-style-guide.md#documentation

Reply:
Done.